components/camel-dynamic-router/src/main/java/org/apache/camel/component/dynamicrouter/DynamicRouterEndpoint.java

```diff
@@ -16,21 +16,26 @@
  */
 package org.apache.camel.component.dynamicrouter;
 
+import java.util.concurrent.ExecutorService;
 import java.util.function.Supplier;
 
-import org.apache.camel.Category;
-import org.apache.camel.Consumer;
-import org.apache.camel.Processor;
-import org.apache.camel.Producer;
+import org.apache.camel.*;
```

Comment:
Just a minor thing: we tend to avoid using star imports.

Reply:
Thanks.  I will change it as requested.

Reply:
I changed these in the source.  I realized that my IDE kept changing it back, so I updated the settings.  It shouldn't happen anymore.